packages/slate-base64-serializer/src/index.js

```diff
@@ -1,5 +1,49 @@
 
 import { State } from 'slate'
+import { Buffer } from 'buffer'
```

Comment:
where does this package come from?

Reply:
this is nodejs's built in Buffer

Reply:
I've used this import for both client and server side and it works.
Is there any scenario that you've found where this import doesn't work?

Reply:
I just can't imagine it is a good idea to import node specific modules to your client bundle, especially if the Browser already defines the imported stuff. Did you check what happens with your client bundle in terms of size?

We could easily have a check for typeof Buffer === 'undefined' and only require the node buffer module in that case.

Reply:
fairPoint, technically speaking this import can be removed since nodejs have Buffer defined globally

Reply:
Whoops, thats true 👍